### PR TITLE
Team page

### PR DESCRIPTION
**Pull request by aslakhellesoy**
This is a fix for #173.

@cucumber/committers - please take 5 minutes to add your own page:

* Click `Create new file` in the [team](https://github.com/cucumber/docs.cucumber.io/tree/team/content/team) directory.
* Add a new file named `yourgithubuser.md` with content similar to [aslakhellesoy.md](https://raw.githubusercontent.com/cucumber/docs.cucumber.io/team/content/team/aslakhellesoy.md)

When we have 10 people we'll merge this. Laggards can add their page later.

### CODE REVIEW

**Comment by mlvandijk** [comment]
@plaindocs @mpkorstanje ?

**Comment by mxygem** [comment]
Is the weight value just a funny characteristic added or is it a formatting thing for the docs?

**Comment by mpkorstanje** [comment]
The profiles are sorted by weight and then by alphabet on the `/team` page.

File: content/team/jaysonesmith.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Jayson Smith
+twitter: [@thatengjayson](https://twitter.com/thatengjayson)
```
Comment (mpkorstanje) [inline content/team/jaysonesmith.md]: Just `thatengjayson` is sufficient.

Comment (mxygem) [reply content/team/jaysonesmith.md]: Beat you to it! 😛

**Comment by mlvandijk** [comment]
Could we sort just by name alphabetically (as the list grows, adjusting weight would be a lot of work...)

**Comment by aslakhellesoy** [comment]
Yeah let’s just remove weight from all team/ pages. Simpler.

**Comment by mxygem** [comment]
@aslakhellesoy I've fixed it for all.

**Comment by xtrasimplicity** [comment]
I don't have a twitter page (that I want to publish, anyway), and noticed that it rendered my display image crazily large. I've updated the page-card template (https://github.com/cucumber/docs.cucumber.io/pull/191/commits/0886ba408aeb465107eccf5d645b88e23288b0ec) to enforce the 80x80 dimension requirement. Let me know if there's a better way. :)

**Comment by xtrasimplicity** [comment]
Also, would you guys be open to adding LinkedIn, for those of us who don't want to publish a twitter profile?

**Comment by aslakhellesoy** [comment]
> Also, would you guys be open to adding LinkedIn, for those of us who don't want to publish a twitter profile?

For sure @xtrasimplicity! Can you find your way how to do that, following the same pattern I used to add twitter support? Both `twitter` and `linkedin` front matter props should be optional. Great if u can add the font-awesome icon too.

**Comment by xtrasimplicity** [comment]
Brilliant, thanks @aslakhellesoy. I'll take a look when I get a chance.

**Pull request merged**